### DISCUSSION TITLE
Google Calendar editing question

### DISCUSSION THREAD

**Discussion by MarkMcDe** [Woundorf/foxreplace#324] [Q&A]
Hey!
This is not an issue with FR, the new version has been working just fine.

However, I need to tap the expertise of the users.
I've been using FR in my Google Calendar to wrap web page tags around certain businesses, eg:
Find: "Afterthought( B[wingery]{3,6}\\b\| Co\\.\| Company\\b){1,2}(?!<)"
inputType | "regexp"
output | "<a href="https://www.facebook.com/afterthoughtbrewing/">Afterthought Brewing Co.</a>"
The tags then appeared in my GCal app as a nice link.
The new version of Google Calendar now has an HTML editor with no plain text option. So running FR on the search term above gives me:
&lt;a href="https://www.facebook.com/afterthoughtbrewing/\">Afterthought Brewing Co.&lt;/a>

My current workaround is to replace the found text with that same text, and the url next to it, to copy and paste into the link menu.

I haven't found any information yet on getting into a plain text version of the new editor, or how to make FR work with an HTML editing UI. I am wondering if anyone else has had better luck.

**Comment by Woundorf** [comment]
I don't have an answer personally, but I'll leave the question open for other users to see.

**Comment by MarkMcDe** [comment]
Thanks so much. Again, there is NO issue with FR. If there was a developer tool like Safari's Snippet editor, I could try to run FR on that window. But for large substitutions, I may need to keep an HTML "sandbox" open that would let me put tags around text, and then copy the formatted text from the display window.
